octue/cloud/pub_sub/service.py

```diff
@@ -341,6 +332,30 @@ def send_delivery_acknowledgment_to_asker(self, topic, timeout=30):
 
         topic.messages_published += 1
 
+    def send_monitoring_update_to_asker(self, data, topic, timeout=30):
```

Comment:
Should be `send_monitor_message`

Note we think that for the future this should belong in a `Child` subclass, since it's meaningless on a `Parent`